api/client/login.go

```diff
@@ -26,7 +26,6 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 
 	flUser := cmd.String([]string{"u", "-username"}, "", "Username")
 	flPassword := cmd.String([]string{"p", "-password"}, "", "Password")
-	flEmail := cmd.String([]string{"e", "-email"}, "", "Email")
```

Comment:
@kencochrane If you add it back, and prefix the flag with a `#`, then the `flag` package should automatically output a deprecation message. See for example; https://github.com/docker/docker/blob/131e2bf12b2e1b3ee31b628a501f96bbb901f479/api/client/ps.go#L28

And, yes, it should be added to the documentation (in the `deprecated.md` file I linked above)